### PR TITLE
Improve code coverage in common module

### LINKED ISSUE #292: Test former common-utils (now in ocean.py)
Follow-up on #226.
The PR #289 moved towards adding common-utils into ocean.py, and the tests already present in the library, but no extra tests were created. We need to add code coverage to this module.

### PR DESCRIPTION
**Pull request by calina-c**
Closes #292.

### CODE REVIEW

**Comment by calina-c** [comment]
Added https://github.com/oceanprotocol/ocean.py/issues/306 as a possible follow-up.

**Review by trentmc — DISMISSED** [review]
Did a cursory review, looks sane

**Review by DMats — CHANGES_REQUESTED** [review]
This PR removes 3 unused methods. I'm fine with this because it reduces maintenance and code bloat. However, it raises questions in my mind about API design cohesion across the Ocean Protocol tech stack. I'd love to discuss this topic further with the team at some point.

File: ocean_lib/common/aquarius/test/test_aquarius.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    assert res.did == ddo.did
+
+    res = aquarius_instance.text_search(ddo.did)
+    assert len(res)
+
+
+def test_multiple_ddos_in_aqua(aquarius_instance):
+    assert aquarius_instance.list_assets()
+    assert aquarius_instance.list_assets_ddo()
+
```
[the remark this answers is not in the document]
Comment (calina-c) [reply ocean_lib/common/aquarius/test/test_aquarius.py:39]: The intention was "test endpoints involving multiple ddos"

File: ocean_lib/common/aquarius/test/test_aquarius.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+def test_ddo_in_aqua(publisher_ocean_instance, metadata, aquarius_instance):
+    publisher = get_publisher_wallet()
+    metadata_copy = metadata.copy()
+
+    ddo = publisher_ocean_instance.assets.create(metadata_copy, publisher)
+    wait_for_ddo(publisher_ocean_instance, ddo.did)
+    aqua_metadata = aquarius_instance.get_asset_metadata(ddo.did)
+
+    # TODO: make sure we need to remove datePublished.
```
[the remark this answers is not in the document]
Comment (calina-c) [reply ocean_lib/common/aquarius/test/test_aquarius.py:24]: That's my question, yes. @trentmc what do you think? Is it the expected behaviour that aqua returns a datePublished key, which is not present in the metadata sample?

File: ocean_lib/common/aquarius/test/test_aquarius.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DMats) [reply ocean_lib/common/aquarius/test/test_aquarius.py:39]: Oh lol.  Then I totally misunderstood.  Please update the test names to be more descriptive and perhaps add docstrings as well.

**Review by DMats — APPROVED** [review]

**Review by DMats — APPROVED** [review]

**Pull request merged**